modelexpress_client/python/modelexpress/vllm_loader.py

```
@@ -0,0 +1,1040 @@
+            _log(f"[TIMING] === RDMA reception complete in {rdma_time:.2f}s ===", "INFO")
+
+            # Now run FP8 processing (transforms weight_scale_inv -> weight_scale)
+            # This will produce IDENTICAL results to source since we have same raw data
+            t0 = _time.perf_counter()
+            _log("[TIMING] Processing weights (FP8 transformation)...", "INFO")
+            process_weights_after_loading(model, model_config, target_device)
+            _log(f"[TIMING] Weight processing complete in {_time.perf_counter() - t0:.2f}s", "INFO")
+
+        total_time = _time.perf_counter() - load_start
+        _log("=" * 60, "INFO")
+        _log(f"[TIMING] MxTargetModelLoader.load_model() COMPLETE", "INFO")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)